ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -0,0 +1,69 @@
+
+let register_smapiv2_server (module S: Storage_interface.Server_impl with type context = unit) sr_ref =
+  let module S = Storage_interface.Server(S) in
+  let rpc = S.process () in
+  let dummy_query_result = Storage_interface.({ driver=""; name=""; description=""; vendor=""; copyright=""; version=""; required_api_version=""; features=[]; configuration=[]; required_cluster_stack=[] }) in
+  Storage_mux.register sr_ref rpc "" dummy_query_result
+
+let override_cbt_functions ?vdi_enable_cbt ?vdi_disable_cbt () =
+  let default a b = match a with
+    | Some a -> a
+    | None -> b
```

Comment:
Or maybe it will be easier to move away from this to a nice upstream option combinator library if I use `Opt` here, so we can just replace all usages of `Opt` from stdext with that library?